loglevel: remove debug switch and add a log level switch

b4ldr opened the pull request:
This PR removes the -D cli argument and adds the new argument `-L`
which allows a user to specify one of none. error, warn, info or debug
for the log level, defaulting to info.

This is my first bit of go coding so apologise if i have made some obvious mitsake

insomniacslk commented:
Thanks @b4ldr ! Can you please also support "none" as we do in https://github.com/coredhcp/coredhcp/blob/master/cmds/coredhcp/main.go#L34 ?

b4ldr commented:
> Thanks @b4ldr ! Can you please also support "none" as we do in https://github.com/coredhcp/coredhcp/blob/master/cmds/coredhcp/main.go#L34 ?

updated, also added fatal and switch to using the same functions as in coreDHCP
